ui/admin/app/templates/scopes/scope/auth-methods/index.hbs

```diff
@@ -174,7 +173,7 @@
           @currentPageSize={{this.pageSize}}
         />
       {{else}}
-        <Hds::ApplicationState data-test-no-auth-method-results as |A|>
```

Comment:
I delete this data-test selector in favour of [this selector](https://github.com/hashicorp/boundary-ui/pull/2654/files#diff-7f512ae43f004e0a490c21b878f15bab1c2304e435e5d7ddb95e0298aafad36eR23). Happy to discuss a revert or other approaches.